app/api/users.py

```diff
@@ -99,12 +101,18 @@ def before_create_object(self, data, view_kwargs):
             raise ConflictException({'pointer': '/data/attributes/email'}, "Email already exists")
 
     def after_create_object(self, user, data, view_kwargs):
+        s = get_serializer()
+        hash = s.dumps([user.email, str_generator()])
+        # link = make_fe_url(path='/email/verify?token={token}'.format(token=hash))
```

Comment:
This method is added into Orders API so if it gets merged I will update here accordingly and if it takes time to review then I will the method definition here in this PR